### ISSUE TITLE
Apache plugin is broken

### ISSUE THREAD

**Issue by Mechazawa** [amonapp/amon-plugins#1]
The apache plugin is currently broken. It either gives no data at all or it throws and exception.

``` js
{
    'versions': {
        'apache': 'Apache',
        'plugin': '1.0.2'
    },
    'error': False,
    'gauges': {},
    'counters': {}
}
```

Using the python module [pyserverstatus](https://pythonhosted.org/pyserverstatus/) should solve the issue.

**Comment by martinrusev** [comment]
Can you please tell me your Apache version + the Distro, so I can take a look

**Comment by Mechazawa** [comment]
Server version: Apache/2.2.22 (Debian)
Server built:   Dec 23 2014 22:48:32

It gives an empty result when the server status url is the default `server-status` instead of `server_status`.

```
Traceback (most recent call last):
  File "/usr/bin/amon-agent.py", line 69, in <module>
    test_plugins()
  File "/usr/local/lib/python2.7/dist-packages/amonagent/check.py", line 173, in test_plugins
    data = plugin.collect() 
  File "/etc/amonagent/plugins/apache/apache.py", line 53, in collect
    key, value = line.split(':')
ValueError: need more than 1 value to unpack
```

`curl 127.0.0.1/server_status?auto`

```
Total Accesses: 4684
Total kBytes: 67211
CPULoad: .0313866
Uptime: 5034
ReqPerSec: .930473
BytesPerSec: 13671.8
BytesPerReq: 14693.4
BusyWorkers: 3
IdleWorkers: 47
Scoreboard: ________________________W......(it goes on for a while)
```

**Comment by Mechazawa** [comment]
After a quick bit of debugging I figured out that the default url returns the html formatted variant of the server status. This is fixed by adding `?auto`. The default url for fetching is also incorrect instead of `server_status` it should be `server-status` as defined in the [status_template](https://github.com/amonapp/amon-plugins/blob/d1628d4b3c010b74121aedd5e5e51642dcd5e3ec/apache/status_template)@d1628d4b3c010b74121aedd5e5e51642dcd5e3ec.

I'm sorry about the hasty look of the OP but I wrote it when I was heading home.
